tests/routers/test_statistics.py

```diff
@@ -0,0 +1,434 @@
+
+    response = client.get("/api/v1/statistics/target/sequence/sequence-type")
+    assert response.status_code == 200
+    assert response.json() == {}
+
+    response = client.get("/api/v1/statistics/target/gene/category")
+    assert response.status_code == 200
+    assert response.json() == {}
+
+    response = client.get("/api/v1/statistics/target/gene/organism")
+    assert response.status_code == 200
+    assert response.json() == {}
+
+    response = client.get("/api/v1/statistics/target/gene/reference")
+    assert response.status_code == 200
+    assert response.json() == {}
+
+    response = client.get("/api/v1/statistics/target/gene/ensembl-identifier")
+    assert response.status_code == 200
+    assert response.json() == {}
+
+    response = client.get("/api/v1/statistics/target/gene/refseq-identifier")
+    assert response.status_code == 200
+    assert response.json() == {}
+
+    response = client.get("/api/v1/statistics/target/gene/uniprot-identifier")
+    assert response.status_code == 200
+    assert response.json() == {}
+
+    response = client.get("/api/v1/statistics/record/experiment/publication-identifiers")
+    assert response.status_code == 200
+    assert response.json() == {}
+
+    response = client.get("/api/v1/statistics/record/experiment/keywords")
+    assert response.status_code == 200
+    assert response.json() == {}
+
+    response = client.get("/api/v1/statistics/record/experiment/doi-identifiers")
+    assert response.status_code == 200
+    assert response.json() == {}
+
+    response = client.get("/api/v1/statistics/record/experiment/raw-read-identifiers")
+    assert response.status_code == 200
+    assert response.json() == {}
+
+    response = client.get("/api/v1/statistics/record/score-set/publication-identifiers")
+    assert response.status_code == 200
+    assert response.json() == {}
+
+    response = client.get("/api/v1/statistics/record/score-set/keywords")
+    assert response.status_code == 200
+    assert response.json() == {}
+
+    response = client.get("/api/v1/statistics/record/score-set/doi-identifiers")
+    assert response.status_code == 200
+    assert response.json() == {}
+
+    response = client.get("/api/v1/statistics/record/score-set/raw-read-identifiers")
+    assert response.status_code == 200
+    assert response.json() == {}
+
+
+## Test target accession statistics
+@pytest.mark.parametrize(
+    "field_value",
+    TARGET_ACCESSION_FIELDS,
+)
+def test_target_accession_statistics(client, setup_router_db, field_value, data_files):
+    """Test target accession statistics endpoint for published score sets."""
+    experiment = create_experiment(client)
+    with patch.object(cdot.hgvs.dataproviders.RESTDataProvider, "_get_transcript", return_value=TEST_CDOT_TRANSCRIPT):
+        score_set = create_acc_score_set_with_variants(client, experiment["urn"], data_files / "scores_acc.csv")
+
+    camelized_field_value = camelize(field_value.replace("-", "_"))
```

Comment:
tests are pretty lightly commented, I'd have one comment here that says, like add one score set in the database, verify that there is one score set containing the parameterized field, blah blah.

never a bad idea to over-comment tests so that when they fail, a developer who has never seen it before will more easily be able to parse out the failure